### PR TITLE
Dynamic data loading updates

### PR DESCRIPTION
**Pull request by dandv**
Pages affected include:

1. http://docs.webix.com/api__atomdataloader_load.html
2. http://docs.webix.com/api__dataloader_ondatarequest_event.html - note #13 
3. http://docs.webix.com/api__refs__atomdataloader.html
4. http://docs.webix.com/api__refs__dataloader.html
5. http://docs.webix.com/api__refs__ui.tree.html
6. http://docs.webix.com/api__link__ui.tree_open.html
7. http://docs.webix.com/api__treedataloader_loadbranch.html
8. http://docs.webix.com/api__treedataloader_ondatarequest_event.html
9. http://docs.webix.com/api__dataloader_ready_config.html
10. http://docs.webix.com/api__link__ui.tree_ready_config.html and all other component `.ready` pages that link to `DataLoader.ready`.

### CODE REVIEW

File: data/api/dataloader_ready_config.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+    webix.message("data");  // displayed first
+    return [
+      { id: "1", value: "Branch", webix_kids: true }
+    ]
+  }(),
+  ready: function() {
+    webix.message("ready");  // displayed second
+    this.data.url = "data/data_dyn.php";  // set the DataStore URL without triggering data loading
+  },
 });
```
Comment (Zwillinge) [inline data/api/dataloader_ready_config.md]: This is not the typical usage of ready and data properties while samples should reflect the typical ones. 
The code you added should be moved to the @descr: section of an article with a comment on data loading.

**Comment by Zwillinge** [comment]
Dan, I strongly appreciate the work you have done, the revision is great.  Still, there are some points that prevent me from merging your pull request. 

1. Please don't capitalize "@short" sections. The descriptions put to "@short" are used for auto-generated "ref" pages like this one http://docs.webix.com/api__refs__dataloader.html. The first capital letters won't fit the existing pattern.  

2. The API article shows the most typical and the most simple samples of code usage. If some use case needs to be provided, it can be put to the "@descr" section while the code there should be enclosed into triple tildes.  I left a comment there. 

2. There's no use to edit "ref"  pages (those featuring "refs" in the path). All of them are auto-generated. They are compiled from "@short" sections of the articles, so it is enough to edit a "@short" part of the related article. 

It would be great if you please to correct the above issues so I can merge this part. Thanks a lot!

**Comment by dandv** [comment]
Thanks for the pointers. I've addressed items 1-3, and made some other corrections.

Can you also please have a look at PR #14 regarding /refs/ pages?

**Comment by Zwillinge** [comment]
Thank you for corrections. There's still one more issue to be aware of - there can be only one unique section per page (`@descr`, `@example`, etc). 
If these sections are repeated, their values will be put into one and the same place and in the worst case, the document will look completely incorrect. 

Here's the correct variant of the file https://github.com/webix-hub/docs/blob/master/data/api/dataloader_ready_config.md

**Pull request merged**